From af428f588f8023784c6f4b0a25d13b70fb7216ab Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Tue, 3 Mar 2020 10:35:40 -0800
Subject: [PATCH] stop using Werror for now

need to work through these warning that only appear on s390x
Werror seems bad for release, makes packaging a nightmare when new
compilers come around
---
 Makefile        | 2 +-
 usr/Makefile    | 2 +-
 usr/initiator.c | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 7e6b734..0069e75 100644
--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@
 DESTDIR ?=
 
 prefix = /usr
-exec_prefix = /
+exec_prefix = /usr
 sbindir = $(exec_prefix)/sbin
 bindir = $(exec_prefix)/bin
 mandir = $(prefix)/share/man
diff --git a/usr/Makefile b/usr/Makefile
index 885243a..1a743d1 100644
--- a/usr/Makefile
+++ b/usr/Makefile
@@ -35,7 +35,7 @@ endif
 PKG_CONFIG = /usr/bin/pkg-config
 
 CFLAGS ?= -O2 -g
-WARNFLAGS ?= -Wall -Wextra -Werror -Wstrict-prototypes -fno-common
+WARNFLAGS ?= -Wall -Wextra -Wstrict-prototypes -fno-common
 CFLAGS += $(WARNFLAGS) -I../include -I. -D_GNU_SOURCE \
 	  -I$(TOPDIR)/libopeniscsiusr -DISNS_ENABLE
 CFLAGS += $(shell $(PKG_CONFIG) --cflags libkmod)
diff --git a/usr/initiator.c b/usr/initiator.c
index 684647c..a5a9d08 100644
--- a/usr/initiator.c
+++ b/usr/initiator.c
@@ -580,7 +580,7 @@ __session_conn_reopen(iscsi_conn_t *conn, queue_task_t *qtask, int do_stop,
 		      int redirected)
 {
 	iscsi_session_t *session = conn->session;
-	uint32_t delay;
+	uint32_t delay = 0;
 
 	log_debug(1, "re-opening session %d (reopen_cnt %d)", session->id,
 			session->reopen_cnt);
-- 
2.26.2

